lib/grep.sh

```diff
@@ -2,7 +2,7 @@
 
 # is x grep argument available?
 function _omb_grep_flag_available {
-  : | grep "$1" "" >/dev/null 2>&1
+  grep "$1" --help >/dev/null 2>&1
```

Comment:
`--help` is not specified by POSIX, so it fails in some other environments. I'd rather suggest the original version before the bug is introduced.

```suggestion
  echo | grep "$1" "" >/dev/null 2>&1
```